### PR TITLE
docs(README.md):  align with docs

### LINKED ISSUE #11707: [docs] update main readme
- wrong npm install
- does not mention other languages
- the capabilities are library oriented. We now have codegen, tracing, test-runner etc.
- redirect users to main getting started on how to run the example tests.
- ...

### PR DESCRIPTION
**Pull request by MarcusFelling**
fixes https://github.com/microsoft/playwright/issues/11707

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ## [Documentation](https://playwright.dev) | [API reference](https://playwright.dev/docs/api/class-playwright/)
 
-Playwright is a framework for Web Testing and Automation. It allows testing [Chromium](https://www.chromium.org/Home), [Firefox](https://www.mozilla.org/en-US/firefox/new/) and [WebKit](https://webkit.org/) with a single API. Playwright is built to enable cross-browser web automation that is **ever-green**, **capable**, **reliable** and **fast**.
+Playwright is a framework for Web Testing and Automation. It allows testing [Chromium](https://www.chromium.org/Home), [Firefox](https://www.mozilla.org/en-US/firefox/new/) and [WebKit](https://webkit.org/) with a single API in [TypeScript](https://playwright.dev/docs/intro), [JavaScript](https://playwright.dev/docs/intro), [Python](https://playwright.dev/python/docs/intro), [.NET](https://playwright.dev/dotnet/docs/intro), [Jav](https://playwright.dev/java/docs/intro). Playwright is built to enable cross-browser web automation that is **ever-green**, **capable**, **reliable** and **fast**.
```
Comment (mxschmitt) [inline README.md:7]:
```suggestion
Playwright is a framework for Web Testing and Automation. It allows testing [Chromium](https://www.chromium.org/Home), [Firefox](https://www.mozilla.org/en-US/firefox/new/) and [WebKit](https://webkit.org/) with a single API in [TypeScript](https://playwright.dev/docs/intro), [JavaScript](https://playwright.dev/docs/intro), [Python](https://playwright.dev/python/docs/intro), [.NET](https://playwright.dev/dotnet/docs/intro), [Java](https://playwright.dev/java/docs/intro). Playwright is built to enable cross-browser web automation that is **ever-green**, **capable**, **reliable** and **fast**.
```

File: README.md [hunk 2]
```diff
@@ -14,32 +14,77 @@ Playwright is a framework for Web Testing and Automation. It allows testing [Chr
...
+
+### Manually
+
+Add dependency and install browsers.
+
+```Shell
+npm i -D @playwright/test
+# install supported browsers
+npx playwright install
 ```
```
Comment (mxschmitt) [inline README.md:43]: At this point we should redirect users to our official docs /docs/intro.

**Review by mxschmitt — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (MarcusFelling) [reply README.md:43]: Had chat offline. Resolving.

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 ## [Documentation](https://playwright.dev) | [API reference](https://playwright.dev/docs/api/class-playwright/)
 
-Playwright is a framework for Web Testing and Automation. It allows testing [Chromium](https://www.chromium.org/Home), [Firefox](https://www.mozilla.org/en-US/firefox/new/) and [WebKit](https://webkit.org/) with a single API. Playwright is built to enable cross-browser web automation that is **ever-green**, **capable**, **reliable** and **fast**.
+Playwright is a framework for Web Testing and Automation. It allows testing [Chromium](https://www.chromium.org/Home), [Firefox](https://www.mozilla.org/en-US/firefox/new/) and [WebKit](https://webkit.org/) with a single API in [TypeScript](https://playwright.dev/docs/intro), [JavaScript](https://playwright.dev/docs/intro), [Python](https://playwright.dev/python/docs/intro), [.NET](https://playwright.dev/dotnet/docs/intro), [Java](https://playwright.dev/java/docs/intro). Playwright is built to enable cross-browser web automation that is **ever-green**, **capable**, **reliable** and **fast**.
```
Comment (pavelfeldman) [inline README.md:7]: Our APIs for languages are different (on purpose). Single API means across browsers, not languages.

Comment (MarcusFelling) [reply README.md]: spoke offline. resolved.

**Pull request merged**